service/storage_proxy.cc

```diff
@@ -4071,6 +4077,12 @@ void storage_proxy::send_to_live_endpoints(storage_proxy::response_id_type respo
     auto& stats = handler_ptr->stats();
     auto& handler = *handler_ptr;
     auto& global_stats = handler._proxy->_global_stats;
+
+    if (handler.get_targets().size() == 0) {
+        handler.no_targets();
+        return;
+    }
+
```

Comment:
The comment above says "wait for hints. Responses count towards consistency.". But we don't wait for hints here.

Reply:
There is nothing to wait for. If there is no targets and hints are enabled there are already stored by this point.

Reply:
Do we have a test that verifies this?

@mlitvk if not please add one. You can use metrics to verify that hints were written.

Reply:
I added a test writing with RF=1 and CL=ANY to a dead node and verifying hints are written